gap_analysis_price/README.rst

```diff
@@ -0,0 +1,79 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+==============
+{gap_analysis_price}
+==============
+
+have price/cost estimation for business requirements.
+
+Installation
+============
+
+To install this module, you need to:
+
+
+
+Configuration
+=============
+
+To configure this module, you need to:
+
```

Comment:
* Create a cost structure in the Gap analysis/Configuration/Cost Structure.
* Set up the cost structure at the CRM opportunity level.